include/glow/Backend/Backend.h

```diff
@@ -147,6 +147,9 @@ class BackendUsingGlowIR : public Backend {
   static RegisterFactory<std::string, FactoryName, Backend>                    \
       FactoryName##_REGISTERED;
 
+/// The backend name used in Glow quantization profiling.
+#define ProfilingBackend "CPU"
```

Comment:
Could you rather use `constexpr char *profilngBackend = "CPU";`?

Reply:
To ensure we don't break builds without the CPU backend enabled, can you make this something like:

```cpp
#ifdef GLOW_WITH_CPU
constexpr char *profilngBackend = "CPU";
#else
constexpr char *profilngBackend = "Interpreter";
#endif
```